src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, hidden = false) {
+    super(name);
+    this.hidden = hidden;
+  }
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
`this.hidden = true`

Reply:
@khozhainov-aleksandr Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores